receiver/prometheusreceiver/metrics_receiver_labels_test.go

```diff
@@ -141,25 +141,42 @@ func TestLabelLimitConfig(t *testing.T) {
 	testComponentCustomConfig(t, targets, mp, cfg)
 }
 
-const targetLabelNameLimit1 = `
+const target1 = `
```

Comment:
```suggestion
const targetLabelLimits1 = `
```

We should keep at least a somewhat descriptive name here.

Reply:
Updated, thanks!